From: Usama Arif <usama.arif@bytedance.com>
Subject: memblock: add missing argument definition
Date: Mon, 18 Sep 2023 10:06:57 +0100

Description of argument memblock_type was missing in the commit that
introduced it.

Link: https://lkml.kernel.org/r/20230918090657.220463-1-usama.arif@bytedance.com
Fixes: 70083e1c8ab4 ("memblock: pass memblock_type to memblock_setclr_flag")
Signed-off-by: Usama Arif <usama.arif@bytedance.com>
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/memblock.c |    1 +
 1 file changed, 1 insertion(+)

--- a/mm/memblock.c~memblock-pass-memblock_type-to-memblock_setclr_flag-fix
+++ a/mm/memblock.c
@@ -892,6 +892,7 @@ int __init_memblock memblock_physmem_add
 
 /**
  * memblock_setclr_flag - set or clear flag for a memory region
+ * @type: memblock type to set/clear flag for
  * @base: base address of the region
  * @size: size of the region
  * @set: set or clear the flag
_
